src/System.Management.Automation/engine/Utils.cs

```diff
@@ -2078,6 +2122,13 @@ public static void SetTestHook(string property, object value)
             }
         }
 
+        /// <summary>This member is used for internal test purposes.</summary>
+        public static object GetTestHookValue(string property)
+        {
+            var fieldInfo = typeof(InternalTestHooks).GetField(property, BindingFlags.Static | BindingFlags.NonPublic);
+            return fieldInfo.GetValue(property);
```

Comment:
Since it's a static field we want, the instance should be `null`

```suggestion
            return fieldInfo.GetValue(null);
```